tests/test_audio_signal.py

```diff
@@ -723,6 +724,9 @@ def test_types(self):
         self.assertIsInstance(signal, np.ndarray)
         self.assertTrue(signal.dtype == np.int16)
         self.assertTrue(type(sample_rate) == int)
+        if sys.version_info[0] == 2:
+            # test unicode string type (Python 2 only)
+            signal, sample_rate = load_audio_file(unicode(sample_file))
```

Comment:
Is there a reason to check this only for Python 2? Shouldn't this be checked for Python 2, too?